sync-core/src/test/java/com/cloudant/sync/datastore/BasicDatastoreConflictsTest.java

```diff
@@ -428,8 +427,8 @@ public void resurrectWinningDocument() throws IOException, ConflictException {
         Assert.assertEquals(deleted.getSequence(), resurrected.getParent());
     }
 
-    @Test(expected = IllegalArgumentException.class)
-    public void resurrectWinningDocumentFails() throws IOException, ConflictException {
+    @Test(expected = DocumentException.class)
```

Comment:
Just a note to make all of these expected exception clauses as exact as possible.